src/app/pages/maps/search-map/search/search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
let's remove it as we don't do unit tests for components (only e2e tests but again not in this project yet)